.golangci.yml

```diff
@@ -45,73 +64,68 @@ issues:
     linters:
     - exhaustivestruct
     - lll
-
-linters-settings:
-  funlen:
-    lines: 110
-    statements: 60
-  staticcheck:
-    go: "1.17"
-  stylecheck:
-    go: "1.17"
-  cyclop:
-    max-complexity: 12
-    skip-tests: true
+  - source: "// .* #\\d+"
+    linters:
+    - godox
+  - path: test/e2e/
+    linters:
+    - goerr113
+    - gomnd
 
 linters:
   disable-all: true
```

Comment:
Should we flip this and set `enable-all: true` so we pick up new linters in the future? And then document any we explicitly disable

Reply:
`enable-all: true` is deprecated

Reply:
enabled all and disabled a selection of linters.